src/Status/StatusCountBar/StatusCountBar.types.ts

```diff
@@ -8,5 +8,5 @@ export type StatusCountBarProps = {
   /**
    * The status count object of different statuses.
    */
-  count: Record<Status, number>;
+  count: Record<Partial<Status>, number>;
```

Comment:
I think you have this type wrong and haven't fixed the issue. Partial around status means nothing here. You need partial around record I think.

See example https://www.typescriptlang.org/play/?#code/C4TwDgpgBAysCGwCuBnKBeKAiA5gezwBMsoAfbAI3mLOyRwBsQsAoF0SKAYTyQDtgGKAAV4AJ2ABLeAwA8AJQgBjPGMKy4iVABoofJAFsKEMQD5TbDtADqYvHxxDFKtbNETpczchSnd+oxMLNhU+FEEVfmAALm5eASEAb1wCYliAZgBfAG4WUPCoAHc7B1jbe0dMZKo0qAAWHKA